_source/handbook/00_the_origin_of_stimulus.md

```diff
@@ -1,5 +1,6 @@
 ---
-permalink: /handbook/origin
+title: The Origin of Stimulus
```

Comment:
Any way to preserve the title-from-heading feature from Jekyll?

Reply:
I might be able to make this happen with a filter. I'll give it a shot.

This is one of the bigger conceptual differences between Jekyll and 11ty. Jekyll allows all sorts of data from all sorts of places. With 11ty, everything goes in the front matter.

Reply:
Made this happen [with a filter](https://github.com/hotwired/stimulus-site/pull/7/files#diff-89d67340237af74d091337b250ea39936bc05e302bd7e10c4f038bb3719fb7df) as well. It's not particularly smart in that it grabs the first element, but it seems fine for this case.

While I was at it, I made sure to allow for front matter titles too. It'll pull from the front matter if there's a value there, and use an auto-generated title if not.